wdp/database/connector_wdp.py

```diff
@@ -0,0 +1,36 @@
+import sqlite3
+
+
+class Database:
+    def __init__(self, database_name="wdp_database.db"):
+        # create connection to database
+        self.connection = sqlite3.connect(database_name)
+        self.cursor = self.connection.cursor()
+
+    def __del__(self):
```

Comment:
Destructors in Python are _not_ reliable.
You'd rather turn this class into a context manager and rewrite this line in `__exit__()` method.
For a more interactive and comprehensible explanation, refer to [this nice video](https://youtu.be/IFjuQmlwXgU?t=248) covering the topic of `__del__` in detail.